README.md

```diff
@@ -106,9 +106,11 @@ DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
 GANDI_API_KEY='api-key'
 GITHUB_ACCESS_TOKEN='github-token'
 WATCH_ORGANIZATION='github-org'
-RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
-RABBITMQ_ADMIN_USERNAME='admin'
-RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_RABBITMQ_API_URL='https://rabbitmq-dev.opencraft.hosting:15671'
+DEFAULT_RABBITMQ_ADMIN_USERNAME='admin'
+DEFAULT_RABBITMQ_ADMIN_PASSWORD='changeme'
+DEFAULT_INSTANCE_RABBITMQ_HOST='rabbitmq-dev.opencraft.hosting'
```

Comment:
I prefer how the [DatabaseServerManager](https://github.com/open-craft/opencraft/blob/9ba290c583012a53c3ec971341bb6ac9f3c55261/instance/models/database_server.py#L60) handles these settings.  You just provide a single `DEFAULT_*_URL` settings variable that is parsed into its protocol, host, port, username, password.

Is that feasible for RabbitMQ too?